lib/Parser/CMakeLists.txt

```
@@ -1,17 +1,23 @@
-add_library (Parser SHARED)
+find_package(BISON REQUIRED)
```

require the same version in cmake as in the file
```suggestion
find_package(BISON 3.2 REQUIRED)
```
Note that on macOS, it fails (2.3 is installed) unless I install a newer bison version with homebrew AND configure cmake with `-DBISON_EXECUTABLE=/opt/homebrew/opt/bison/bin/bison`